test/bigtest/interactors/InvoiceFormInteractor.js

```diff
@@ -40,7 +40,7 @@ import { OptionListInteractor } from '@folio/stripes-acq-components/test/bigtest
 export default interactor(class InvoiceForm {
   static defaultScope = '#pane-invoice-form';
   isLoaded = isPresent('[class*=paneTitleLabel---]');
-  saveButton = new Interactor('[data-test-button-save-invoice]');
```

Comment:
imo it's better to keep our selectors

Reply:
I see, `id` is defined as a prop

imo formfooter component should have interactor